docker/ubuntu/README.md

```diff
@@ -86,3 +86,22 @@ $ docker build \
 $ docker run -it grass-py3-pdal:latest-ubuntu /bin/bash
 bash-5.0#
 ```
+
+## Test the docker image
+
+Note: Adjust the volume mount to the path of the GRASS GIS source code directory.
+
+```bash
+# Test basic functionality
+$ docker run -ti \
+         -v /opt/src/grass:/grassdb \
```

Comment:
In my understanding `/grassdb` is the (mounted) destination in the container where the grass data is stored, not the source code. I would distinguish between data and software and use a different target directory. In the end it is not really that important but it might prevent confusion.